gecko_client/health_types.go

```diff
@@ -1,20 +1,22 @@
 package gecko_client
 
+import "github.com/ava-labs/gecko/api/health"
+
 type Check struct {
-	Message map[string]interface{} 	`json:"message"`
-	Timestamp string	`json:"timestamp"`
-	Duration int 	`json:"duration"`
-	ContiguousFailures int 	`json:"contiguousFailures"`
-	TimeOfFirstFailure string	`json:"timeOfFirstFailure"`
+	Message            map[string]interface{} `json:"message"`
+	Timestamp          string                 `json:"timestamp"`
+	Duration           int                    `json:"duration"`
+	ContiguousFailures int                    `json:"contiguousFailures"`
+	TimeOfFirstFailure string                 `json:"timeOfFirstFailure"`
 }
 
 type LivenessInfo struct {
-	Checks map[string]Check	`json:"checks"`
-	Healthy bool	`json:"healthy"`
+	Checks  map[string]Check `json:"checks"`
+	Healthy bool             `json:"healthy"`
 }
 
 type GetLivenessResponse struct {
-	JsonRpcVersion string       `json:"jsonrpc"`
-	Result         LivenessInfo `json:"result"`
-	Id             int          `json:"id"`
+	JsonRpcVersion string                  `json:"jsonrpc"`
+	Result         health.GetLivenessReply `json:"result"`
```

Comment:
Noice - way better to use Gecko's inbuilt type. Ideally, this client would get completely transitioned into the Gecko repo

Reply:
Absolutely